openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -264,6 +271,35 @@ def certificate_data(self):
         if self.enrollment_object:
             return get_cert_data(self.effective_user, self.course, self.enrollment_object.mode)
 
+    @property
+    def entrance_exam_data(self):
```

Comment:
Feels like you are wasting one of the benefits of consolidating these fields in a dict named `entrance_exam_data` by still prefixing every field in that dict with `entrance_exam_`.  :) Not worth changing - just saying.

Reply:
Yeah, I think I'm going to leave alone just because when we deconstruct in the frontend, it helps to have a field called `entrance_exam_enabled` rather than just `enabled`. Is it verbose? Absolutely. But maybe it helps some poor soul down the line.